api/controllers/auth_controller.py

```diff
@@ -77,10 +80,17 @@ def get_users_list(_) -> dict:
 
 @cross_origin()
 @admin_required
-def add_user(_) -> dict:
+@validate(body = AuthRequestModel)
```

Comment:
Sync with `master`. Pydantics is now supported by the project, you can just change this to:
```suggestion
@validate(body = Auth)
```

All Schemas can be removed